common/log/tag/tags.go

```diff
@@ -47,6 +47,11 @@ func ClusterName(clusterName string) Tag {
 	return newStringTag("cluster-name", clusterName)
 }
 
+// ActiveClusterName returns tag for ActiveClusterName.
+func ActiveClusterName(activeClusterName string) Tag {
+	return newStringTag("cluster-name", activeClusterName)
```

Comment:
Should this be active cluster name ?
```suggestion
	return newStringTag("active-cluster-name", activeClusterName)
```